style-guide/src/components/handover-process/ResourceBox.js

```diff
@@ -4,12 +4,12 @@ import copy from '../../../static/handover-process';
 import styles from '../../styles/handover.module.scss';
 
 export default ({ resource }) => (
-  <div className={styles.resource}>
+  <div className={styles.resource} onClick={() => { window.open(copy[`${resource}.link`], '_blank')}}>
```

Comment:
Can this be either an anchor tag or a button? It'd be more semantically correct than having a click handler on a presentational element